Jenkinsfile

```diff
@@ -431,6 +512,179 @@ pipeline {
+                        //                 powershell "npm run test -- brave_browser_tests ${BUILD_TYPE} --output brave_browser_tests.xml"
+                        //                 xunit([GoogleTest(deleteOutputFiles: true, failIfNotNew: true, pattern: "src/brave_browser_tests.xml", skipNoTestFiles: false, stopProcessingIfError: true)])
+                        //             }
+                        //             catch (ex) {
+                        //                 currentBuild.result = "UNSTABLE"
+                        //             }
+                        //         }
+                        //     }
+                        // }
+                        stage("dist") {
+                            steps {
+                                powershell """
+                                    Import-PfxCertificate -FilePath \"${KEY_PFX_PATH}\" -CertStoreLocation "Cert:\\LocalMachine\\My" -Password (ConvertTo-SecureString -String \"${AUTHENTICODE_PASSWORD_UNESCAPED}\" -AsPlaintext -Force)
+                                    npm run create_dist -- ${BUILD_TYPE} --channel=${CHANNEL} --debug_build=false --official_build=true
+                                """
+                                powershell '(Get-Content src\\brave\\vendor\\omaha\\omaha\\hammer-brave.bat) | % { $_ -replace "10.0.15063.0\", "" } | Set-Content src\\brave\\vendor\\omaha\\omaha\\hammer-brave.bat'
+                                powershell "npm run create_dist -- ${BUILD_TYPE} --channel=${CHANNEL} --build_omaha --tag_ap=x86-${CHANNEL} --target_arch=ia32 --debug_build=false --official_build=true"
+                            }
+                        }
+                        // TODO: add upload step here
+                        stage("github_upload") {
+                            when {
+                                expression { env.LABEL_SUFFIX == 'release' }
+                            }
+                            steps {
+                                withCredentials([[
+                                    $class: 'AmazonWebServicesCredentialsBinding',
+                                    credentialsId: 'brave-browser-binaries-upload',
+                                    accessKeyVariable: 'BRAVE_S3_ACCESS_KEY',
+                                    secretKeyVariable: 'BRAVE_S3_SECRET_KEY'
+                                ]]) {
+                                    powershell 'npm run upload -- --target_arch=ia32'
+                                }
+                            }
+                        }
+                        stage("archive") {
+                            steps {
+                                // commented because it takes much longer to copy to Jenkins thant to S3
+                                // archiveArtifacts artifacts: "${OUT_DIR}/BraveBrowser*.exe", fingerprint: true
+                                s3Upload(acl: "Private", bucket: "${BRAVE_ARTIFACTS_BUCKET}", includePathPattern: "brave_installer*.exe",
```

Comment:
might have to put specific installer file name here for 32-bit

Reply:
Both win64 and win32 installer files are called `brave-installer.exe`, which was incorrect in the current pattern match. This meant that the 64 bit installer was not being archived to the s3 bucket for artifacts in previous builds.

Reply:
the 64-bit one that gets uploaded is `brave_installer_72_0_62_0.exe` so we need that wildcard